pkg/util/certmanager/certmanager.go

```diff
@@ -67,15 +67,18 @@ func NewYurttunnelServerCertManager(
 	_ = wait.PollUntil(5*time.Second, func() (bool, error) {
 		dnsNames, ips, err = serveraddr.GetYurttunelServerDNSandIP(clientset)
 		if err != nil {
+			klog.Errorf("get yurt-tunnel-service info fail, %v", err)
```

Comment:
Just an advice: `failed get yurt tunnel server dns and ip` will be more accurate, because there's no svc calld  `yurt-tunnel-service`.